### PR TITLE
Update inputgroup component, added postfix again....

### PR DESCRIPTION
**Pull request by eskilsen**
Update snapshot

## Description

Something was wrong with the visual rep of postfix example in inputfield, this is now fixed. 
## How Has This Been Tested?

Ran all tests, update snapshots
## Screenshots (if appropriate):
image
to
image

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have updated the **CHANGELOG** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

## Review instructions
[Review instructions](../REVIEW_INSTRUCTIONS.md)

### CODE REVIEW

File: src/App/components/InputGroup/index.js [hunk 1]
```diff
@@ -98,7 +94,10 @@ const InputGroup = ({
                                 ))}{"\n\t"}
                             </select>
                             :
-                            <input {...attrs} />}
+                            <>
+                                <input {...attrs} />
+                                {postfix ? <Addon type={type} value={postfixValue} disabled={disabled} postfix={postfix} /> : null }{prefixValue ? "\n" : null}
```
Comment (goldenraphti) [inline src/App/components/InputGroup/index.js:99]: @eskilsen ,
it seems like the bug got introduced between release 10.0.0 & 10.1.0 , which means it most likely comes from this commit: https://github.com/SwedbankPay/design.swedbankpay.com/commit/d4624895ca1f14442c4cc94640525b73309f45ee by its removal of line 103

Now, from my **very** limited knowledge of the codebase 😅 , I'm seeing we're bringing back the `<Addon ...>`  for postfix, but a _slightly_ different version of it. For instance:
- before the removal it was `type={postfixType}` and it is now `type={type}`
- `color` attribute is also removed

Basically:

**before**: `{postfixValue ? <Addon type={postfixType} value={postfixValue} color={postfixBtnColor} disabled={disabled} postfix={postfix} /> : null }{postfixValue ? "\n" : null}</>}`

**after**:  `{postfix ? <Addon type={type} value={postfixValue} disabled={disabled} postfix={postfix} /> : null }{prefixValue ? "\n" : null}`

I have no clue what is right or wrong, and no clue what is this `type` doing yet 😅 , but I just wanted to bring it up to confirm the changes are intentional 👍

Comment (eskilsen) [reply src/App/components/InputGroup/index.js:99]: We can discuss this further in a huddle on Slack 👍

**Review by goldenraphti — APPROVED** [review]
